### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by kubitre**
this is summary request for issues: #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,22 +10,61 @@ hocon current configuration configuration library
 - environment variables
 - default values from tag in structures
```
Comment (kubitre) [inline README.md]: etstset

File: pipeline/not_value.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package pipeline
+
+// NotAValue - value for check setuping in fields
```
Comment (artemkaxboy) [inline pipeline/not_value.go]: If you are using such a word there must be double `setupping`. But anyway there is no such a word, `setting up` fits better.

File: pipeline/pipeline.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+	// Needed confgiure structure
+	structure interface{},
+	// filename for file configuring
+	fileName string,
+	pipelineChanes []FuncType) (err error) {
+	pipeline := getFunctionChain(pipelineChanes)
+	currentChain := pipeline.chains
+	for {
+		fmt.Println("Level: Debug. current stage function.")
+		defer func() {
```
Comment (artemkaxboy) [inline pipeline/pipeline.go]: You should not define defer in a cycle

File: pipeline/pipeline.go [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+		return nil, errors.New(notSupportedTypeError + "configure server configurator source. Not realized yet")
+	default:
+		return nil, errors.New(notSupportedTypeError +
+			"you should search in lib available type configurator source")
+	}
+}
+
+// Configure - main configurer
+func Configure(
+	// Needed confgiure structure
```
Comment (artemkaxboy) [inline pipeline/pipeline.go]: typo: confgiure

File: pipeline/pipeline.go [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+	structure interface{},
+	// filename for file configuring
+	fileName string,
+	pipelineChanes []FuncType) (err error) {
```
Comment (artemkaxboy) [inline pipeline/pipeline.go]: typo: pipelineChaines

File: README.md [hunk 6]
```diff
@@ -1,4 +1,6 @@
 # Go_structor [![Actions Status](https://github.com/goreflect/go_structor/workflows/CI_dev/badge.svg)](https://github.com/goreflect/go_structor/actions?query=workflow%3ACI_dev) [![Go Report Card](https://goreportcard.com/badge/github.com/goreflect/go_structor)](https://goreportcard.com/report/github.com/goreflect/go_structor)
```
Comment (artemkaxboy) [inline README.md]: Change repo name everywhere if it's been renamed

File: go.sum [hunk 7]
```diff
@@ -1,4 +1,5 @@
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/goreflect/go_structor v0.0.0-20200116205716-48742d4c2b6e h1:MRl+84ypcf8lvNTlxPHU7PZxdTfBEqzQODgjJC3p5n0=
```
Comment (artemkaxboy) [inline go.sum]: don't need it

File: pipeline/pipeline.go (hunk 4 above)
Comment (kubitre) [reply pipeline/pipeline.go]: ?

File: pipeline/not_value.go (hunk 2 above)
Comment (kubitre) [reply pipeline/not_value.go]: ++

File: pipeline/pipeline.go (hunk 3 above)
Comment (kubitre) [reply pipeline/pipeline.go]: ++

File: pipeline/pipeline.go (hunk 4 above)
Comment (kubitre) [reply pipeline/pipeline.go]: ++

File: pipeline/pipeline.go (hunk 5 above)
Comment (kubitre) [reply pipeline/pipeline.go]: ++

File: go.sum (hunk 7 above)
Comment (kubitre) [reply go.sum]: fixed but this file does not affect assembly

File: README.md (hunk 6 above)
Comment (kubitre) [reply README.md]: ++

Comment (artemkaxboy) [reply README.md]: Some more places
image

**Comment by artemkaxboy** [comment]
Are we going to use squash when merging?

**Comment by kubitre** [comment]
> Are we going to use squash when merging?

no, it’s better not to connect to fully follow the story. We will delete branches

File: README.md (hunk 6 above)
Comment (kubitre) [reply README.md]: done

**Pull request merged**